docs/search/views/create-search-views.md

```diff
@@ -0,0 +1,26 @@
+---
+sidebar_position: 10
+title: Create a Search View
+---
+
+This page explains how to create a new search view.
+
+## Create Search View
+
+Follow these instructions to create a new search view using the GDN console web UI.
+
+1. [Log in to your Macrometa account](https://auth.paas.macrometa.io/).
+1. Click **Search**.
+1. Click **New View**.
+1. Enter information about the view and then click **Create**.
+
+   - **Name -** The user-defined name for the search view.
+   - **Mapping Definition -** - Optional. Choose existing data from which to create an index.
+    - **Collection -** Select an existing collection.
+    - **Field -** Enter a field name for the selected collection.
+    - **Analyzer -** Select a text analyzer or identity analyzer to break up search inputs for improved searching and sorting.
+   - **Primary Sort -** Optional. The sorting order for each attribute. Cannot be changed after view is created.
+    - **Field -** Specify the sorting accodring to collections and fields in the mapping definition.
+    - **Direction -** Set the sorting order to ascending (default) or descending.
+
+After creating a view, you can **Rename** or **Delete** it from the **Search** screen.
```

Comment:
Looks like we can also do this from the CLI: https://github.com/Macrometacorp/docs/blob/master/docs/cli/search-views-cli.md, which means we can probably do it from the API and maybe code as well.

Please file a Jira with the ImportTechDebt label to research and add these later to both Create Search Views and Edit Search Views.

What about viewing search views and deleting search views as a task?